README.md

```diff
@@ -16,6 +16,7 @@ The following plugins are installed in the docker image:
 1. fluent-plugin-hostname
 1. fluent-plugin-retag
 1. fluent-plugin-kinesis
```

Comment:
nah:
(a) we want to run both versions of the plugin for a while ourselves (dev vs prod)
(b) the instructions say to use 'latest' ( :( ), so we might break others if we remove